src/get-started/codelab.md

```diff
@@ -743,6 +812,12 @@ where you add the following functionality:
   containing the saved favorites.
 * Modify the theme color, making an all-white app.
 
+## Explore the Flutter SDK
+
+* <a href="https://docs.flutter.dev/get-started/flutter-for/react-native-devs">Flutter for React Native developers</a>
```

Comment:
Was this change intended? Seems unrelated to the rest of the change.

Reply:
Build is green! We should be good to land once this comment is resolved.

Reply:
I'd revert this part of the change as it's an unrelated change. If it's required, please fold into a new PR. Alternatively, it might be guff from Git history shenanigans...